AndroidNativeKotlinTemplate/app/build.gradle

```diff
@@ -1,14 +1,12 @@
 apply plugin: 'com.android.application'
-apply plugin: 'kotlin-android'
```

Comment:
We don't need this plugin anymore. It's natively supported in `Android Studio 3.0`.